ui/pages/settings/flask/snaps-list-tab/snap-list-tab.js

```diff
@@ -58,19 +50,13 @@ const SnapListTab = () => {
               return (
                 <SnapSettingsCard
                   className="snap-settings-card"
-                  isEnabled={snap.enabled}
                   key={key}
-                  onToggle={() => {
-                    onToggle(snap);
-                  }}
-                  description={snap.manifest.description}
-                  url={snap.id}
+                  packageName={removeSnapIdPrefix(snap.id)}
                   name={snap.manifest.proposedName}
```

Comment:
Please use `getSnapName`

Reply:
Updated to use `getSnapName` function. I had to do some refactoring to get metadata since `getSnapName` requires it and if it's not passed to the function, it starts having unexpected behavior with snap names. Let me know if there is anything else.